includes/sqp/uhd-radarr-resolution.md

```diff
@@ -1,7 +1,7 @@
 ??? abstract "Resolution - [Click to show/hide]"
-    | Custom Format                                                                                                                | Score                                      | Trash ID                                |
-    | ---------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------ | --------------------------------------- |
-    | [{{ radarr['cf']['1080p']['name'] }}](https://raw.githubusercontent.com/TRaSH-/Guides/master/docs/json/radarr/cf/1080p.json) | {{ radarr['cf']['1080p']['trash_score'] }} | {{ radarr['cf']['1080p']['trash_id'] }} |
-    | [{{ radarr['cf']['2160p']['name'] }}](https://raw.githubusercontent.com/TRaSH-/Guides/master/docs/json/radarr/cf/2160p.json) | :warning: 151 :warning:                    | {{ radarr['cf']['2160p']['trash_id'] }} |
+    | Custom Format                                                                                                                |                                  Score                                   | Trash ID                                |
+    | ---------------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------: | --------------------------------------- |
+    | [{{ radarr['cf']['1080p']['name'] }}](https://raw.githubusercontent.com/TRaSH-/Guides/master/docs/json/radarr/cf/1080p.json) |          {{ radarr['cf']['1080p']['trash_scores']['default'] }}          | {{ radarr['cf']['1080p']['trash_id'] }} |
```

Comment:
same as all other guides. should we have a redundant score even if only default exists to minimize impact? this can be done as a phase 2.

Reply:
Agree we should discuss under the new GHI as a phase two, so I'll mark this one as resolved.